gradle/verification-metadata.xml

```diff
@@ -2734,6 +2734,12 @@
             <sha256 value="2a4e84a51defe9108579b3c0a86bb41e54f04e9042e83adf4348a974dcf1dee6" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="io.github.java-diff-utils" name="java-diff-utils" version="4.10">
```

Comment:
I pointed out the same thing: https://github.com/JetBrains/kotlin/pull/4348#discussion_r626823062, but the below two are needed?

Reply:
Yes, these are needed when you have new dependencies (see `build.gradle.kts` files). Otherwise you will get a build failure.